turf/rhumb_distance/_rhumb_distance.py

```diff
@@ -0,0 +1,89 @@
+from typing import Dict, List
+
+import numpy as np
+
+from turf.helpers import convert_length, degrees_to_radians
+from turf.helpers import earth_radius
+from turf.invariant import get_coord
+
+
+def rhumb_distance(origin, destination, options: Dict = None) -> float:
+    """
+    Calculates the rhumb distance between two Points. Units are defined in helpers._units
+
+    # https://en.wikipedia.org/wiki/Rhumb_line
+
+    :param start: starting point [lng, lat] or Point feature
+    :param end: ending point [lng, lat] or Point feature
+    :param options: dictionary with units as an attribute.
+                    Units are defined in helpers._units
+
+    :return: distance between the 2 points
+
+    """
+    kwargs = {}
+    if isinstance(options, dict) and "units" in options:
+        kwargs.update(options)
```

Comment:
since we're not passing `kwargs` to any function, I would just check if options is a dict:

```python
if not isinstance(options, dict):
    options = {}
```

and then on line 40 get `units` from `options`:

```python
distance = convert_length(
    distance_in_meters, "meters", options.get("units", "kilometers")
)
```

Reply:
resolved. 👍